index.html

```diff
@@ -0,0 +1,487 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <link rel="stylesheet" type="text/css" href="./Assets/stylesheet.css">
+        <title>Space Ripples Reveal Big Bang's Smoking Gun</title>
+    </head>
+    <body>
+            <div class="aside">
+                    <span><strong>Editor's pick</strong></span>
+                    <hr>
+                    <div>
+                        <img src="./Assets/31SCI-SATELLITES2-threeByTwoSmallAt2X.jpg" alt="A link to the past">
+                        <p>Anarchy in the U.K</p>
+                    </div>
+                    <div>
+                        <img src="./Assets/31SCI-SATELLITES2-threeByTwoSmallAt2X.jpg" alt="A link to the past">
+                        <p>Anarchy in the U.K</p>
+                    </div>
+                    <div>
+                        <img src="./Assets/31SCI-SATELLITES2-threeByTwoSmallAt2X.jpg" alt="A link to the past">
+                        <p>Anarchy in the U.K</p>
+                    </div>
+                    <div>
+                        <img src="./Assets/31SCI-SATELLITES2-threeByTwoSmallAt2X.jpg" alt="A link to the past">
+                        <p>Anarchy in the U.K</p>
+                    </div>
+                </div>
+                <div class="aside-lower">
+                    <h3><strong>Sign Up for Science Times</strong></h3>
+                    <p>We’ll bring you stories that capture the wonders of the human body, nature and the cosmos.</p>
+                    <button class="subscribe-btn">Sign Up</button>
+                </div>
+        <header>
+            <h1>
+                <span>Space Ripples Reveal</span> 
+                <span>Big Bang's Smoking Gun</span>
+            </h1>
+            <div class="cover">
+                <img class="cover-position" src="./Assets/18cosmos-superJumbo.jpg" alt="Alan Guth"><br>
+                <span class="cover-position subtext">Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the<br>
+                universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</span><br>
+                <span class="credit cover-position">Rick Friedman for the New York Times</span>
+                
+            </div>    
+        </header>
+        <div class="content">
```

Comment:
Use `<main>` instead of `<div>`. Div tag has no semantic value. It is important for many reasons to make your HTML code semantic. Check out this article to learn more about it: https://www.lifewire.com/why-use-semantic-html-3468271